js_modules/dagit/packages/ui/src/components/TokenizingField.tsx

```diff
@@ -191,7 +192,10 @@ export const TokenizingField: React.FC<TokenizingFieldProps> = ({
     // If suggestions are present, autoselect the first one so the user can press
     // enter to complete their search. (Esc + enter is how you enter your raw text.)
     if (!active && suggestions.length) {
-      setActive({text: suggestions[0].text, idx: 0});
+      const item = suggestions[0];
```

Comment:
For things like this, will the suggestions.length > 0 on the previous line automatically remove the unwrap warning here?

Reply:
Discussed in Slack, but the `length` check is not sufficient to guarantee a value at this index. We need to be explicit with all array access.

It's a bit more annoying to do it, but it adds a nice level of strictness that we currently don't have. :)